docs/TEMPLATE.md

```diff
@@ -1,4 +1,9 @@
 ---
+title: The title of the post, page, or document
+description: A short description of the page's content
+image: URL to an image associated with the post, page, or document (e.g., /assets/page-pic.jpg)
```

Comment:
I see you don't provide `image` in the SmartScan article? Either make `image` optional, add an image in the SmartScan article, or get rid of it?

Reply:
I'm going to remove it since we don't have an image for SmartScan yet. Also, I updated those variables as optional in the README.md